Nadim.CinemaReservationSystem.Web/ClientApp/src/components/Cinema/FormGeneralCinemaInfo.js

```diff
@@ -0,0 +1,206 @@
+                    <h4>
+                        <strong>City : </strong>
+                    </h4> 
+                    <input 
+                        type="text" 
+                        className="form-control form-control-sm" 
+                        value={this.state.city} 
+                        onChange={this.handleCityChange}
+                        placeholder="City"
+                    />
+                </div>
+                <div
+                    className={this.state.displayedComponents.name ? '' : 'hidden'}
+                >
+                    <h4>
+                        <strong>Cinema name : </strong>
+                    </h4> 
+                    <input 
+                        type="text" 
+                        className="form-control form-control-sm" 
+                        value={this.state.name} 
+                        onChange={this.handleNameChange}
+                        placeholder="Name"
+                    />
+                </div>
+                <div
+                    className={this.state.displayedComponents.cinemaRoomsCount ? '' : 'hidden'}
+                >
+                    <h4>
+                        <strong>Number of cinema rooms : </strong>
+                    </h4> 
+                    <input 
+                        type="text" 
+                        className="form-control form-control-sm" 
+                        value={this.state.cinemaRoomsCount} 
+                        onChange={this.handleCinemaRoomsChange}
+                        placeholder="Rooms"
+                    />
+                </div>
+                <div
+                    className={this.state.displayedComponents.defaultSeatPrice ? '' : 'hidden'}
+                >
+                    <h4>
+                        <strong>Price for default seat USD : </strong>
+                    </h4> 
+                    <input
+                        type="text" 
+                        className="form-control form-control-sm" 
+                        value={this.state.defaultPrice} 
+                        onChange={this.handleDefaultPriceChange}
+                        placeholder="Price"
+                    />
+                </div>
+                <div
+                    className={this.state.displayedComponents.vipSeatPrice ? '' : 'hidden'}
+                >
+                    <h4>
+                        <strong>Price for VIP seat USD : </strong>
+                    </h4> 
+                    <input
+                        type="text" 
+                        className="form-control form-control-sm" 
+                        value={this.state.vipPrice} 
+                        onChange={this.handleVipPriceChange}
+                        placeholder="Price"
+                    />
+                </div>
+                <Button 
+                    bsStyle="primary"
+                    onClick={this.handleSubmitCinemaInfoClick} 
+                    disabled={
```

Comment:
Here and on every form - as a user how can I tell why the "Submit" button is disabled? What should I do to correct my mistakes?